webui/react/src/pages/WorkspaceDetails/WorkspaceMembers.tsx

```diff
@@ -67,7 +67,7 @@ const GroupOrMemberActionDropdown: React.FC<GroupOrMemberActionDropdownProps> =
 
   const menuItems: DropDownProps['menu'] = useMemo(() => {
     const MenuKey = {
-      Remove: 'remove',
+      Remove: 'Remove',
```

Comment:
i made the change in[ the different PR](
https://github.com/determined-ai/determined/pull/5640/files#diff-a9a21494cf20b899ce8dc5a2baa98d4b53f28afe1d3ea1ab63d30895b43b14b3R84)

and i noticed this change breaks the code

Reply:
I agree, your PR fixes this in the correct place (label) and my change here doesn't make sense. I'll get a review to you on that PR soon and then work on getting that in first